core/src/main/scala/org/apache/spark/sql/delta/commands/OptimizeTableCommand.scala

```diff
@@ -57,24 +61,75 @@ abstract class OptimizeTableCommandBase extends RunnableCommand with DeltaComman
 case class OptimizeTableCommand(
     path: Option[String],
     tableId: Option[TableIdentifier],
-    partitionPredicate: Option[String])
+    partitionPredicate: Option[String])(val zOrderBy: Option[Seq[UnresolvedAttribute]])
```

Comment:
why have Option[Seqp[]] instead of just Seq[]? any difference between None and Seq.empty here